helpers/deploymentConfig.ts

```diff
@@ -162,6 +174,22 @@ export const getGlobalConfig: () => Promise<NetworkConfig> = async () => {
           borrowCap: convertToUnit(4_400_000, 18),
           vTokenReceiver: vTreasuryAddress,
         },
+        {
+          name: "Venus TWT",
+          asset: "TWT",
+          symbol: "vTWT",
+          rateModel: InterestRateModels.JumpRate.toString(),
+          baseRatePerYear: convertToUnit("0.02", 18),
+          multiplierPerYear: convertToUnit("0.2", 18),
+          jumpMultiplierPerYear: convertToUnit("3", 18),
+          kink_: convertToUnit("0.5", 18),
+          collateralFactor: convertToUnit("0.5", 18),
+          reserveFactor: convertToUnit("0.25", 18),
+          initialSupply: convertToUnit(5_000, 18),
```

Comment:
Is initialSupply different from mainnet? [reference](https://github.com/VenusProtocol/venus-protocol/pull/527/files#diff-18185c86bd8baed36466d7a325efd11e7729ffdd43200a4808db76a6340185ffR237)